spec/lrama/diagnostics_spec.rb

```diff
@@ -29,6 +29,7 @@
           %token tEQEQ  "=="
 
           %type <i> class /* comment for class */
+          %type string_1 string_2 /* missing `%type` tag */
```

Comment:
What this comment means? I think `%type` is not missed on this grammar file.

Reply:
What I meant to say there is that the `%type` tag is missing. In other words, I wanted to make it clear that I was testing the lack of `<tag>`.

Reply:
I got it. What do you think about comment like `/* <tag> is omitted */` ?

Reply:
Thank you! I updated comment.